shakecast/app/eventprocessing.py

```diff
@@ -106,6 +106,14 @@ def create_new_event_notifications(groups, event, scenario=False):
 def create_inspection_notifications(groups, shakemap, scenario=False):
     notifications = []
     for group in groups:
+        # check new_event magnitude to make sure the group wants a
+        # notificaiton
+
+        event_spec = group.get_new_event_spec(scenario=scenario)
```

Comment:
For inspection notifications, we determine whether or not to send based on the impact criteria. A group with a green impact threshold shouldn't get notifications for events that don't impact facilities at least to that level, but they should get that notification even if it wasn't large enough to trigger a new event. Definitely interested in hearing your thinking on this one though

Reply:
DWR is trying to switch over to only potential impact emails rather than separate event and potential impact e-mails. We made the switch to notifications that meet the green impact threshold. However, we've gotten feedback from our users that they would still like to receive e-mails for lower thresholds of shaking. By adding in an event threshold to the potential impacts notifications  and making changes to the workbook and notification template (added dummy sites to the workbook at the grey level and some conditional statements to the notification template) our users now receive one e-mail notification for any earthquake that exceeds our magnitude threshold OR meets the green impact criteria.